src/test/java/org/opensearch/security/util/RepeatRule.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.security.util;
```

Comment:
Opting to keep this code in the PR, but I will remove it upon request.

As long as we are on JUnit4, this will allow us to annotate individual tests to run n times by adding:

```
@Rule
public RepeatRule repeatRule = new RepeatRule();

@Test
@Repeat(10)
public void testDelayInSecurityIndexInitialization() throws Exception {
    ...
}
```

Reply:
I like it for local 'stress' testing, lets keep it!  How would you feel about updating the testing section of our docs to include this?

Reply:
I updated the DEVELOPER_GUIDE to add this in :). IMO our DEVELOPER_GUIDE could use some more work to instruct people on how to run different kinds of tests in this repo.